### ISSUE TITLE
End of season summary

### ISSUE THREAD

**Issue by mdryden** [110yards/fantasy#91]
This is a two parter:

1) I'd like there to be a "final summary" page that shows up once the is finished.  It should highlight the champion, runner up, and the regular season standings.

Super artistic rendering:

![image](https://user-images.githubusercontent.com/5611490/141658225-46dcb0cf-100c-4ab6-8e8e-cafe2b3d4e6b.png)

2) I want this view to be rendered from a "season summary" object which can be created at the end of each season.  Leagues that continue from season to season should be able to view a history of their past seasons.

**Comment by kamikazecanuck** [comment]
That would be cool

**Comment by mdryden** [comment]
I've already kind of started on this one, it's definitely going to happen.

A secondary piece (which might not be ready for 2022), is that I hope to allow commissioners to manually enter past seasons, if they'd like to capture some history prior to 2021.

**Comment by mdryden** [comment]
The first part of this is done now, you should see the end of season summary when you view your league now.

The next part I'll open another issue for, but it will be to allow commissioners to create these manually, and to view a list of past results.  I think this will work well for leagues that come back year after year.

**Comment by mdryden** [comment]
Oops, i see a small bug; the standings aren't ordered properly.  I'll fix next chance I get.
